ts-test/src/App.tsx

```
@@ -0,0 +1,42 @@
+        <Button
+          onClick={() => setCount((count) => count + 1)}>
+          count is {count}
+        </Button>
+        <p>
+          Edit <code>src/App.tsx</code> and save to test HMR
+        </p>
+      </div >
+      <p className="read-the-docs">
+        Click on the Vite and React logos to learn more
+      </p>
+      <CardWithForm></CardWithForm>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)